tests/test_capsule/test_capsule_operations.py

```diff
@@ -28,43 +35,52 @@ def test_capsule_creation(alices_keys):
 
 
 def test_capsule_equality():
-    one_capsule = Capsule(point_e=Point.gen_rand(),
+    params = default_params()
```

Comment:
We pass this in a ton. Should we make this a pytest fixture?

Reply:
Yes.

Reply:
Yes, we should. Perhaps in another PR?